cinema/views.py

```diff
@@ -0,0 +1,59 @@
+from django.shortcuts import get_object_or_404
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+from rest_framework import status
+
+from cinema.models import Movie
+from cinema.serializers import MovieSerializer
+
+
+@api_view(["GET", "POST"])
+def movie_list(request):
+    if request.method == "GET":
+        movies = Movie.objects.all()
+        serializer = MovieSerializer(movies, many=True)
+        return Response(
+            serializer.data,
+            status=status.HTTP_200_OK
+        )
+    if request.method == "POST":
+        serializer = MovieSerializer(data=request.data)
+        if serializer.is_valid():
+            serializer.save()
+            return Response(
+                serializer.data,
+                status=status.HTTP_201_CREATED
+            )
+        return Response(
+            serializer.errors,
+            status=status.HTTP_400_BAD_REQUEST
+        )
+
+
+@api_view(["GET", "PUT", "DELETE"])
+def movie_detail(request, pk):
+    current_movie = get_object_or_404(Movie, pk=pk)
+
+    if request.method == "GET":
+        serializer = MovieSerializer(current_movie)
+        return Response(
+            serializer.data,
+            status=status.HTTP_200_OK
+        )
+    elif request.method == "PUT":
+        serializer = MovieSerializer(Movie, data=request.data)
+        if serializer.is_valid():
+            serializer.save()
+            return Response(
+                serializer.data
```

Comment:
explicit is better than implicit.
Despite 200 status is returned by default, it's better to write it